src/values/color.rs

```diff
@@ -370,24 +370,30 @@ impl ToCss for CssColor {
           // If the #rrggbbaa syntax is not supported by the browser targets, output rgba()
           if let Some(targets) = dest.targets {
             if !Feature::CssRrggbbaa.is_compatible(targets) {
```

Comment:
@devongovett Maybe it would be better to convert only when `dest.minify = true` here too? I don't really like the parser conversion format if the CSS is not compressed. What do you think?

Reply:
Sure